linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_encrypt_sent_records/ansible/shared.yml

```diff
@@ -10,4 +10,5 @@
     line: enable_krb5 = yes
     regexp: ^\s*enable_krb5\s*=\s*.*$
     state: present
+    mode: 0644
```

Comment:
I have found that this file has mode 0640 by default on RHEL7. Reporter of issue #4845  also wants 0640. If you change it to 0644 it will be reported by `rpm -Va`.

Reply:
Actually @jan-cerny that was a brain/keyboard typo. It is set now to the correct permissions.